Good-Bids/goodbids
# [#399, #414, #422] Webpack fix, Auction Wizard Start, and Auction WIzard Product

# Summary

1. We're replacing Vite with Webpack for all assets in `./src`
2. We're adding the start for the auction wizard
3. We're adding the "create reward product" screen for the auction wizard

## Issues

- #399
- #414
- #422
- #442

## Testing Instructions

1. `cd client-mu-plugins/goodbids && npm install && npm run build`
2. Click around the app and see if all the JS is still working.
3. Go to the site admin and find `Auction Wizard`
4. View the auction wizard start screen, click the button to go to product creation page
5. Define a reward product
6. Click "Save and Continue" and see either errors on fields you missed or a placeholder screen for auction creation.

## Screenshots

![image](https://github.com/Good-Bids/goodbids/assets/48295174/d3ca8bc5-3135-46fe-a987-c29546ffefcf)

![image](https://github.com/Good-Bids/goodbids/assets/48295174/7c49431c-37b0-4ac7-93ec-2706ec7b8339)

![image](https://github.com/Good-Bids/goodbids/assets/48295174/9f369079-e019-486a-a0c3-ced4b9977026)

![image](https://github.com/Good-Bids/goodbids/assets/48295174/95b40ba8-658c-4a93-9417-02e1bf99a745)

![image](https://github.com/Good-Bids/goodbids/assets/48295174/5b1c2928-462a-4f3b-aa72-b0085f41c5aa)
